examples/anthos-service-mesh-multicluster/mysql-docker/mysql-test.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: mysql-test
+  labels:
+    app: mysql-test
+spec:
+  ports:
+  - port: 5000
+    name: http
+  selector:
+    app: mysql-test
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mysql-test
+  labels:
+    version: v1
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: mysql-test
+      version: v1
+  template:
+    metadata:
+      labels:
+        app: mysql-test
+        version: v1
+    spec:
+      containers:
+      - name: mysql-test
+        image: liaojianhe/mysql-test:v1.0.0
```

Comment:
what is this image? it doesn't seem like an image google controls.

Reply:
That's last-name-first-name of my collaborator, jianhel@google.com